pytrip/utils/slicedos.py

```diff
@@ -0,0 +1,211 @@
+from pytrip import dos
+from pytrip import ctx
+import argparse
+import logging
+from numpy import arange, meshgrid, ma
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import os
+import sys
+
+
+def check_compatible(a, b):
+    """ Simple comparison of cubes. if X,Y,Z dims are the same, and
+    pixel sizes as well, then they are compatible. (Duck typed)
+    """
+    eps = 1e-5
+
+    if a.dimx != b.dimx:
+        logging.error("DIMX does not match: "+str(a.dimx)+" "+str(b.dimx))
```

Comment:
Better to create logger instance (`logging.getLogger`) instead of using module methods, see: http://victorlin.me/posts/2012/08/26/good-logging-practice-in-python